src/Sylius/Bundle/AdminBundle/Tests/Action/Account/RequestPasswordResetActionTest.php

```diff
@@ -0,0 +1,313 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Tests\Action\Account;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\AdminBundle\Action\Account\RequestPasswordResetAction;
+use Sylius\Bundle\AdminBundle\Form\Model\PasswordResetRequest;
+use Sylius\Bundle\AdminBundle\Form\RequestPasswordResetType;
+use Sylius\Bundle\CoreBundle\Command\Admin\Account\RequestResetPasswordEmail;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Twig\Environment;
+
+final class RequestPasswordResetActionTest extends TestCase
+{
+    private FormFactoryInterface&MockObject $formFactoryMock;
```

Comment:
I would be for not using `Mock` suffix, but it is not a blocker for the PR 🖖🏻 
```suggestion
    private FormFactoryInterface&MockObject $formFactory;
```